### PR TITLE
Update README to the recent changes

### PR DESCRIPTION
**Pull request by jakubfijalkowski**

### CODE REVIEW

**Comment by marcin-dardzinski** [comment]
Should https://github.com/leancodepl/corelibrary/blob/v9.0-preview/docs/tests/integration_tests/index.md#integration-test should also be updated to use `IntegrationFact`, plus some invocation magic?

**Comment by jakubfijalkowski** [comment]
@marcin-dardzinski I don't think so. These attributes are in the `test/LeanCode.Test.Helpers` project, so no project outside this repo can see it.

**Review by lukaszgarstecki — APPROVED** [review]

**Comment by marcin-dardzinski** [comment]
Okay, I was mistaken by the project name. So, maybe outside of this repo, but what are the guidelines for the projects? Are we changing anything there?

**Comment by jakubfijalkowski** [comment]
@marcin-dardzinski nope, nothing has changed IMO, at least in the context of _this_ repository. ExampleApp is slightly different beast.

**Comment by jakubfijalkowski** [comment]
Well, maybe one thing - the examples should adjust the interface of `IAsyncLifetime`.

**Review by marcin-dardzinski — APPROVED** [review]

File: docs/tests/integration_tests/index.md [hunk 1]
```diff
@@ -241,7 +232,7 @@ public class UnauthenticatedExampleAppTestApp : ExampleAppTestApp
     public HttpCommandsExecutor Command { get; private set; } = default!;
     public HttpOperationsExecutor Operation { get; private set; } = default!;
 
-    public override async Task InitializeAsync()
+    public override async ValueTaskTask InitializeAsync()
```
Comment (Patonymous) [inline docs/tests/integration_tests/index.md]:
```suggestion
    public override async ValueTask InitializeAsync()
```
:)

**Pull request merged**